### ISSUE TITLE
[DEV] Content progression tool (season switch)

### ISSUE THREAD

**Issue by FrancescoBorzi** [azerothcore/forum#124]
AzerothCore supports the 3.3.5a client and game content, however it's quite common to change the DB in order to close/open contents and create some kind of progression. For example, some servers start with Naxxramas, then they change season and open Ulduar, etc... PvP vendors should be updated too.

I have the feeling that all people doing this are working over and over on the same thing, which is a waste of time.

So why don't we create a repository with all the SQL code needed to switch from one season to another?

**Comment by BarbzYHOOL** [comment]
Yes and they all do it the wrong way.

The best way is to create game events like the arena seasons, but it's more work but it's clean and "professional", just a hint for others

**Comment by Yehonal** [comment]
No, it's not enough, because there are some "UPDATES" queries on things
that are in-game...it's not just a matter of things that are spawned or
not.

The easiest solution is a collection of "migration queries" with which you
can change the version of the database....it's still a non-super-blizzlike
solution but doable.

In the end, also the blizzard started the official new classic experience
with the features of 1.12 client but the content of 1.1.0 I guess. So it's
a reasonable way to go.

If you think about, no one would play with old non-fixed features (think
about all bugs of 3.0.2 that have been fixed for example), but everyone
want to revive the progression and the content, the important thing is that
can still be called Wotlk.

That's why I think the database migration are the best way

**Comment by Yehonal** [comment]
However, with the project ChromieCraft we will work on it
